### ISSUE TITLE
Display controller variable in custom widget title

### ISSUE THREAD

**Issue by benoit-nadaud** [angular-dashboard-framework/angular-dashboard-framework#239]
Is the any way to access a variable define in the controller of the widget ?
I'm trying to display the number of elements displayed in the widget content.

Thanks for ADF which is very cool and well done.

**Comment by sdorra** [comment]
No, this is not possible a controller is able to manage the widget content and not the title.
